opcua/ua/ua_binary.py

```diff
@@ -503,6 +503,9 @@ def struct_from_binary(objtype, data):
             if not test_bit(val, idx):
                 continue
         val = from_binary(uatype, data)
+        if val is None and uatype[:6] == 'ListOf':
+            val = []
+            
```

Comment:
What is the issue exactly?the UA protocols allows to return Null values which are best translated to None in Python. So the original code looked correct

Reply:
Because the empty lists are iterated in the more high level modules. A non iterable NoneType
leads to an error there. See issue #1127. This error occurred when trying to communicate with a Softing server.

For example:
File "C:_repo\opcua-client-gui-master\venv\lib\site-packages\opcua\tools.py", line 423, in application_to_strings
for url in app.DiscoveryUrls:
TypeError: 'NoneType' object is not iterable

Instead of checking the lists against None everywhere I chose to fix this issue on its roots.